playbooks/droplet.yml

```diff
@@ -9,13 +9,12 @@
     public_key: "{{ lookup('ansible.builtin.file', ansible_env['HOME'] ~ '/.ssh/sammy.key.pub') }}"
 
   tasks:
-
     - name: Create SSH key
       digitalocean.cloud.ssh_key:
         state: present
         token: "{{ digitalocean_token }}"
         public_key: "{{ public_key }}"
-        name: "sammy"
+        name: sammy
```

Comment:
The quotes on this really fail linting?!

Reply:
So interestingly, that one didn't fail linting, but running `ansible-lint --fix` auto-did that. 🤷

Reply:
Why are they making fixes for things they aren't flagging as a linting violation? Is it a violation that didn't populate in the results? 🤷 Ansible tooling is half-madness